### ISSUE TITLE
Support full SQL templates

### ISSUE THREAD

**Issue by janblok** [9tigerio/db2rest#836]
While according to the docs there is support for named queries: (=SQL template containing a select statement)
https://db2rest.com/docs/namedsql/introduction-to-named-queries

Its seems not possible to execute a create, update or delete "template" SQL statement, from a configuration file.

**Comment by kdhrubo** [comment]
@nikhilnc1 yes - someone needs to check the tests and write docs to bring this closure @ntananh worked on this. @ntananh can you help @nikhilnc1 i can mark this as a good first issue.

**Comment by kdhrubo** [comment]
@nikhilnc1 are you working on this ?

**Comment by janblok** [comment]
Closed by PR  https://github.com/9tigerio/db2rest/pull/933
